src/java/org/apache/cassandra/utils/JMXServerUtils.java

```diff
@@ -149,6 +155,55 @@ public static JMXConnectorServer createJMXServer(int port, boolean local) throws
         return createJMXServer(port, null, local);
     }
 
+    private static void setJmxRemoteSslPropertiesFromFile(String remoteSslConfigFile)
+    throws IOException, FileNotFoundException
+    {
+        Path remoteSslConfigFilePath = Paths.get(remoteSslConfigFile);
+
+        if (!Files.exists(remoteSslConfigFilePath))
+        {
+            throw new FileNotFoundException(
+                "JMX remote SSL configuration file does not exist: " + remoteSslConfigFilePath
+            );
+        }
+
+        try (final BufferedReader bufferedReader = Files.newBufferedReader(remoteSslConfigFilePath))
+        {
+            logger.info("Reading JMX remote SSL configuration file: {}", remoteSslConfigFilePath);
+            for(String rawLine; (rawLine = bufferedReader.readLine()) != null;)
+            {
+                final String line = rawLine.trim();
+                if (line.startsWith("#") || line.isEmpty())
+                    continue;
+
+                String[] parts = line.split("=");
+                if (parts.length != 2)
+                {
+                    logger.warn(
+                        "JMX remote SSL configuration file {} contains a property in an unknown format; " +
+                        "skipping property: {}", remoteSslConfigFilePath, line);
+                    continue;
+                }
+
+                if (!parts[0].startsWith("javax.net.ssl"))
+                {
+                    logger.warn(
+                        "JMX remote SSL configuration file {} must only contain javax.net.ssl properties; " +
+                        "skipping property: {}", remoteSslConfigFilePath, parts[0]);
+                    continue;
+                }
```

Comment:
I don't think we should implement our own parsing for properties files, when `java.util.Properties.load(java.io.Reader)` can do that according to the known specification.